Posit conf 2023 summary (#235)
rladies/rladies.github.io: content/news/2023-10-09-positconf-chicago-meetup/index.en.md

easier for people to attend events.

*Angela:* Online events allow for the possibility of collaborations with
other chapters that didn't exist before. Additionally, before there were
a lot of details in terms of having to find space, get pizza, and open
doors, now with Zoom this is less so the case now and it makes things
easier on the organizers and those people that are coming to speak.

## What is the process of starting a chapter?

*Riva:* Send an email to [chapters\@rladies.org](mailto:chapters@rladies.org). A member of the Onboarding Team will welcome you
and explain process. Chapters are permitted for cities, but not for a
university or company. If you wish to have a thematic meet-up, you could
approach a chapter like R-Ladies Remote and they can help create
thematic meet-ups.

## How would I step up and take over an inactive local chapter?

*Riva:* Same process; email
[chapters\@rladies.org](mailto:chapters@rladies.org), and they would be
happy to begin the process to reactivate your chapter.

## If I don't have a local chapter, is there a way to join a "virtual" chapter? Or are there chapters that regularly meet online?

> The R-Ladies Remote chapter serves this purpose. Additionally, many

Author: We don't have a speaker listed here - Mo or Riva?

Reviewer: I think this was both Mo and Riva who spoke on this question.

> The R-Ladies Remote chapter serves this purpose. Additionally, many

Reviewer 2: ```suggestion
*Riva*: The R-Ladies Remote chapter serves this purpose. Additionally, many
```

approach a chapter like R-Ladies Remote and they can help create
thematic meet-ups.

## How would I step up and take over an inactive local chapter?

*Riva:* Same process; email
[chapters\@rladies.org](mailto:chapters@rladies.org), and they would be
happy to begin the process to reactivate your chapter.

## If I don't have a local chapter, is there a way to join a "virtual" chapter? Or are there chapters that regularly meet online?

The R-Ladies Remote chapter serves this purpose. Additionally, many
chapters continue to have virtual events and many welcome participants
outside of their geographic location; they are often announced in the
events channel of the Community Slack channel or on Meet-up.

## Have you considered expanding to include Python?

*Riva:* Stephanie and I are also PyLadies organizers, and we are now planning to host joint R-Ladies and
PyLadies events together in Valparaíso. Cosima Meyer (Cologne organiser, Germany) also organizes joint events
in Germany, and it is a great way to expand the way you are reaching
people.

## Does R-Ladies raise funding for events?

*Yani:* R-Ladies receives money from institutional and individuals

> donors; R Consortium is also a large donor and that mostly goes to

Reviewer 3: ```suggestion
donors; R Consortium is our main sponsor because R-Ladies is a Top Level project, this money mostly goes to
```

Meetup fees. For R-Ladies chapters there is a different process -
chapters can apply for R Consortium grants. If you do not want to manage
money you can ask for non-monetary donations, like a venue, pizza,

> drinks, etc. More recently some chapters have use Open Collective to

Reviewer 3: ```suggestion
drinks, speaker travel expenses, child care services, etc. More recently some chapters have use Open Collective to
```
